tools/rvn/CommandLineApp.cs

```diff
@@ -645,6 +728,41 @@ private static void ValidateSetupOptions(CreateSetupPackageParameters parameters
             }
         }
 
+
+        private static void ValidateHelmValuesPath(CreateSetupPackageParameters parameters)
+        {
+            if (string.IsNullOrWhiteSpace(parameters.HelmValuesOutputPath))
+            {
+                throw new InvalidOperationException("Please provide a valid file name for the helm values yaml.");
+            }
+            
+            if (Path.HasExtension(parameters.HelmValuesOutputPath) == false)
+            {
+                parameters.HelmValuesOutputPath += ".yaml";
+            }
+            else if (Path.GetExtension(parameters.HelmValuesOutputPath)?.Equals(".yaml", StringComparison.OrdinalIgnoreCase) == false)
```

Comment:
Are you sure Path.GetExtension() is not going to return null? Maybe look for something more bulletproof.